providers/ttys.rb

```diff
@@ -0,0 +1,151 @@
+#
+# Cookbook Name:: gecos-ws-mgmt
+# Provider:: ttys
+#
+# Copyright 2013, Junta de Andalucia
+# http://www.juntadeandalucia.es/
+#
+# All rights reserved - EUPL License V 1.1
+# http://www.osor.eu/eupl
+#
+
+V2 = ["GECOS V2","Gecos V2 Lite"]
+regex1 = "tty[1-6].conf"
+regex2 = ".*(tty[1-6]).conf(.bak)?"
+upstart_dir ="/etc/init/"
+backup_suffix = ".bak"
+
+action :setup do
+  begin
+    if new_resource.support_os.include?($gecos_os)
+
+      Chef::Log.debug("ttys: #{new_resource.ttys}")
+
+      if new_resource.ttys # ENABLE TTYs
```

Comment:
This comment (and another in the else section) are wrong.

As far as I can see, this 'if' section is DISABLING TTYs

Reply:
Done.